scripts/prettier/prettier-helpers.js

```diff
@@ -30,6 +45,15 @@ function getPrettierBinary() {
 function runPrettier(files, config = {}) {
   const { check, logErrorsOnly, runAsync } = config;
 
+  const fileIsGlob = files.length === 1 && files[0].includes('*');
```

Comment:
to give you better context:

our custom `format` script is executed only in 2 modes

- `all` which runs on whole monorepo 
  - this will create always  deep glob, so 
    -   `files === <some-path>/**/*.{<Extensions>}`
- `affected files` which runs only on affected files
  - this will pass all files by relative path (now filtered out by their extension that is supported by prettier)
    - it will never include `*` char 
    - EXAMPLE: `files === [./packages/foo-bar/index.js, ./rfcs/some-doc.md]`

Reply:
Ok that makes much more sense, thanks for clarifying!